doc/application_workflow/private_registry.md

```diff
@@ -0,0 +1,102 @@
+# Running a private registry with VIC
+
+In this example, we will run a private Docker registry on VIC and push and pull images using VIC.  VMWare also offers an enterprise ready registry named [Harbor](https://github.com/vmware/harbor) that can be used in place of the base Docker registry.
+
+## Prerequisite
+
+Before going through this example, we need to re-emphasize some concepts around Docker and VIC.  The following examples are shown using Linux.  For Windows and Mac users, these examples should not differ much.
+
+Installing VIC will also require installing Docker locally.  When Docker is installed, we will get both a client (CLI or command line interface) and a daemon that handles all local container operations.  Local containers are those that run on the user local machine instead of a VMWare vSphere/ESXi environment.  The CLI is important as it will be most user's touchpoint for working with containers on VIC and on their local system.  The distinction between using the CLI against both environment is very important in this example.  By default, the CLI will use the local Docker daemon.  After setting some environment variables, the CLI can be instructed to send all operations to VIC instead of the local Docker daemon.  The two environment variable are DOCKER_HOST and DOCKER_API_VERSION.
+
+In this example, we are deploying an insecure registry with no authentication for simplicity.  We will also be targeting an ESXi environment.
+
+## Workflow
+
+In terminal #1: (local Docker)
+
+1. Open a terminal and make sure that it will use the local Docker daemon.  At a command prompt, issue  
+```
+    $> unset DOCKER_HOST
+```
+2. Install a VCH for a private registry using vic-machine
+3. Run Docker's registry on the first VCH
```

Comment:
A link to the docs for Docker's registry will be helpful. Perhaps https://docs.docker.com/registry/deploying?